lib/ui/hooks.dart

```diff
@@ -263,6 +260,9 @@ void Function(Uri) _getHttpConnectionHookClosure(bool mayInsecurelyConnectToAllD
         // or if connection is to `https`, allow the connection.
         return;
       }
+      if (_isLoopback(uri.host)) {
```

Comment:
Perhaps update the comment at line 257 since it won't throw if uri is a loopback uri